packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.tsx

```diff
@@ -223,7 +248,7 @@ const LockedFunctionSettings = (props: Props) => {
                     tag="label"
                     style={[styles.dropdownLabel, styles.domainMin]}
```

Comment:
Does the label component for domain min also need an aria label like the domain max one?

Reply:
No, it's automatically labeled by the label itself. You can see in the test file that the aria labels successfully get set as `"domain min"` and `"domain max"`.